docker/wedlocks.dockerfile

```diff
@@ -2,7 +2,6 @@ FROM unlock-core
 
 # Dependencies for wedlocks
 RUN mkdir /home/unlock/wedlocks
-COPY --chown=node wedlocks/package-lock.json /home/unlock/wedlocks/.
```

Comment:
Is this based on an older version of master? The one that went in yesterday should have changed all of these to `yarn.lock`

Reply:
hum. I think it's rebased :/